devdoc/DeveloperDocs.md

```diff
@@ -2,25 +2,123 @@
 
 This is a collection of documentation by developers for developers.
 
-## Introduction
+## Introduction to Developer Documentation
 
-Welcome to the Developer Documentation! This guide serves as a comprehensive resource for developers working with the HPCC Platform. Here, you'll find detailed information about various components, tools, and services to help you build, maintain, and optimize your projects effectively.
+Welcome to the Developer Documentation! This guide serves as a comprehensive resource for developers working with the HPCC Systems Platform. Here, you'll find detailed information about various components, tools, and services to help you build, maintain, and optimize your projects effectively.
 
-## How to Use This Documentation
+### How to Use This Documentation
 
-- **Explore Components**: Navigate through the sections below to learn about specific components of the HPCC Platform.
+- **Explore Components**: Navigate through the sections below to learn about specific components of the HPCC Systems Platform.
 - **Get Started Quickly**: Refer to the introductory guides and examples provided in each section.
 - **Contribute**: If you have insights or improvements, feel free to contribute to this documentation.
 
-## Platform Versions
+### Developer/Contributor Guidelines
```

Comment:
Is this section not needed, or is the description missing?

Reply:
> Is this section not needed, or is the description missing?

I combined Version Support into Developer/Contributor Guidelines